icubam/predicu/data.py

```diff
@@ -86,7 +85,11 @@ def load_if_not_cached(data_source, cached_data, **kwargs):
     )
   if cached_data is None or data_source not in cached_data:
     load_data_fun_signature = inspect.signature(load_data_fun)
-    kwargs = {}
+    kwargs = {
+      k: v
+      for k, v in kwargs.items()
+      if k in load_data_fun_signature.parameters
+    }
```

Comment:
Sorry it was me removing this. Why is it necessary. Functions will be called with defaults values anyway?

Reply:
`api_key` and `icubam_host` were not propagatade to `load_combined_bedcounts_public`